### PR TITLE
feat:adds AvatarGroup

### PR DESCRIPTION
**Pull request by stutirao**
### What does this implement/fix? Explain your changes.

...

### Does this close any currently open issues?

...

### Any other comments?

...

### Dependent PRs/Commits

...

### Describe breaking changes, if any.

...

### Checklist

Check all those that are applicable and complete.

- [ ] Merged with latest `master` branch

### CODE REVIEW

File: core/components/atoms/avatarGroup/AvatarGroup.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    appearance: appearances[6]
+  }] : [];
+  const newList = list.slice(0, max);
+  const List = newList.concat(hiddenAvatars);
+
+  return (
+    <div {...baseProps} className={AvatarGroupClass}>
+      {List.map((item, ind) => {
+        const { appearance, firstName, lastName } = item;
+        let tooltip = (`${firstName ? firstName : ''}${' '}${lastName ? lastName : ''}`);
```
Comment (riyalohia) [inline core/components/atoms/avatarGroup/AvatarGroup.tsx]: Space in a string can be added as: str = '${str1} ${str2}'.

File: core/components/atoms/avatarGroup/AvatarGroup.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+  const AvatarGroupClass = classNames({
+    ['AvatarGroup']: true,
+  }, className);
+  let str = '';
+  list.map((listItem, ind) => {
+    const strNew = ind >= max ? (`${listItem.firstName ? listItem.firstName : ''}
+    ${listItem.lastName ? listItem.lastName : ''}`) : '';
+    str = `${str}${' '}${strNew}`;
+  });
```
Comment (riyalohia) [inline core/components/atoms/avatarGroup/AvatarGroup.tsx]: Loop should run for ind > max only. Also, strNew is not required.

File: css/src/components/avatarGroup.css [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+.AvatarGroup-item:last-child {
+    margin-right: 0;
+}
+.GhostBg{
+    background-color: white;
+  width: 32px;
+  height: 32px;
+  padding: 3px ;
+  margin-left:-14px;
+  border-radius:19px;
```
Comment (riyalohia) [inline css/src/components/avatarGroup.css]: Use CSS variables.

**Review by riyalohia — CHANGES_REQUESTED** [review]

File: core/components/atoms/avatarGroup/AvatarGroup.tsx [hunk 4]
```diff
@@ -0,0 +1,71 @@
+import * as React from 'react';
+import { Avatar, Appearance } from '../avatar/Avatar';
+import { BaseProps, extractBaseProps } from '@/utils/types';
+import Tooltip from '@/components/atoms/tooltip';
+const appearances: Appearance[] = ['primary', 'alert', 'warning', 'success',
```
Comment (aditya-kumawat) [inline core/components/atoms/avatarGroup/AvatarGroup.tsx]: Not required. Import from Avatar.

File: core/components/atoms/avatarGroup/AvatarGroup.tsx [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+                  firstName={firstName}
+                  lastName={lastName}
+                />
+              </Tooltip>
+            </span>
+          </div>
+        );
+      })}
+      {extraAvatars ? (
+        <div className="OuterBox" >
```
Comment (aditya-kumawat) [inline core/components/atoms/avatarGroup/AvatarGroup.tsx]: Not as per BEM convention.

**Pull request closed without merging**